src/content/reference/app-configuration/index.md

```diff
@@ -0,0 +1,280 @@
+---
+title: App Configuration
+description: Documentation on the various levels of App configuration and how they get merged into a final Values object.
+date: 2020-04-02
+type: subsection
+---
+
+# App Configuration Reference
+
+Giant Swarm's App Catalog allows you to easily install Apps across your entire
+fleet of clusters. Currently, Apps are packaged as Helm charts. By configuring
+your App you set the `Values` that become available to the templates when they
+are deployed.
+
+There are three levels of configuration:
+
+`catalog`, `cluster`, and `user`.
+
+Each level of configuration has two types of values that you can provide:
+
+`config-values` and `secret-values`.
+
+All values are gathered together and merged into the final `Values`
+object that will become available to the Chart template.
+
+Values are merged in the following order:
+
+1. `catalog-config-values`
+2. `catalog-secret-values`
+3. `cluster-config-values`
+4. `cluster-secret-values`
+5. `user-config-values`
+6. `user-secret-values`
+
+If no value is provided then the default in the Chart's `values.yaml` is used.
+
+As a user of the App Catalog you will most likely not come into contact with
+the `catalog` and `cluster` level of configuration. Our interfaces make it
+easiest to set `user` level configuration values.
```

Comment:
I think this needs to be stronger. For customers they should only edit the user values.

In the case of default apps if the user changes the catalog or cluster level config it will be overwritten by the operators. But I guess in the future if customers are deploying their own apps via App Catalog then they can change all 3 levels of config.

Reply:
I'll add a sentence to make it clearer that as a user, the catalog and cluster level should basically be off limits for you, and nothing something for you to think about. 

But for a full reference of how it all works, and in the future for App or AppCatalog developers, it's useful to have all levels mentioned.

Reply:
Thanks for the update and yes I agree the docs should cover all 3 levels.